manim/mobject/types/opengl_vectorized_mobject.py

```diff
@@ -218,7 +219,7 @@ def get_style(self):
             "shadow": self.get_shadow(),
         }
 
-    def match_style(self, vmobject, recurse=True):
+    def match_style(self, vmobject, recurse=True, family=True):
```

Comment:
Why is `family` added if it's seemingly not used?

Reply:
I think this was from an early approach of trying to make `DashedVMobject` compatible, I've changed the strategy now to use recurse or family depending on `config.renderer`.

Reply:
b80e6b35a259e58f8b122e23a8cf380d3a2c4044